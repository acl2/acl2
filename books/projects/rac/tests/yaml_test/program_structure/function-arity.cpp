// RAC begin

int foo(int a) { return a; }

int bar() { return foo(); }

// RAC end
