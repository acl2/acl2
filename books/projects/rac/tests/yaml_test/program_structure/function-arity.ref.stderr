function-arity.cpp:5 (19-24): Expected 1 argument(s), got 0.
  5 | int bar() { return foo(); }
    |                    ^^^^^
note: Defined here: 
  3 | int foo(int a) { return a; }
    | ^^^^^^^^^^^^^^
